test/integration_test.tsx

```diff
@@ -761,39 +779,64 @@ describe('integration tests', () => {
       assert.equal(wrapper.text(), 'Example');
     });
 
-    it('supports simulating events on Components (simulateEventsOnComponents: true)', () => {
-      function FancyButton({ onClick, children }: any) {
-        return (
-          <button type="button" onClick={onClick}>
-            {children}
-          </button>
-        );
-      }
+    describe('simulateEventsOnComponents: true', () => {
+      setAdapter(() => new Adapter({ simulateEventsOnComponents: true }));
 
-      function App() {
-        const [count, setCount] = useState(0);
+      it('supports simulating events on Components', () => {
+        function FancyButton({ onClick, children }: any) {
+          return (
+            <button type="button" onClick={onClick}>
+              {children}
+            </button>
+          );
+        }
 
-        return (
-          <div>
-            <div id="count">Count: {count}</div>
-            <FancyButton onClick={() => setCount(count + 1)}>
-              Increment
-            </FancyButton>
-          </div>
-        );
-      }
+        function App() {
+          const [count, setCount] = useState(0);
+
+          return (
+            <div>
+              <div id="count">Count: {count}</div>
+              <FancyButton onClick={() => setCount(count + 1)}>
+                Increment
+              </FancyButton>
+            </div>
+          );
+        }
 
-      const wrapper = shallow(<App />, {
-        adapter: new Adapter({ simulateEventsOnComponents: true }),
-      });
-      assert.equal(wrapper.find('#count').text(), 'Count: 0');
+        const wrapper = shallow(<App />);
+        assert.equal(wrapper.find('#count').text(), 'Count: 0');
 
-      wrapper.find(FancyButton).simulate('click');
-      assert.equal(wrapper.find('#count').text(), 'Count: 1');
+        wrapper.find(FancyButton).simulate('click');
+        assert.equal(wrapper.find('#count').text(), 'Count: 1');
+      });
     });
   });
 
   describe('"string" rendering', () => {
     addStaticTests(renderToString as any);
+
+    describe('useRenderToString: true', () => {
```

Comment:
The name of this describe block needs to be updated (eg. "using preact-render-to-string")

Reply:
Thanks! Updated